test/unit/puppetlabs/services/config/puppet_server_config_service_test.clj

```diff
@@ -18,12 +18,17 @@
   [puppet-server-config-service jruby-puppet-pooled-service jetty9-service
    profiler/puppet-profiler-service])
 
+(def required-config
```

Comment:
Man that's the second time I've caused that same problem in the last 3 days.  :)  nice catch.